platform-sdk/platform-apps/demos/HashgraphDemo/build.gradle.kts

```diff
@@ -14,6 +14,9 @@
  * limitations under the License.
  */
 
-plugins { id("com.hedera.hashgraph.application") }
+plugins {
+    id("com.hedera.hashgraph.application")
+    id("com.hedera.hashgraph.mock-release-tasks")
```

Comment:
I think it would be better if we would somehow unify this in `com.hedera.hashgraph.conventions` and sort out projects that publish vs that don't more centrally. But for now, if this fixes things, lets do it like this I would say.

Reply:
This was the shortest path to a fix. Unfortunately the current project structure does not really provide a clear separation of unpublished vs. published projects.